db/schema.rb

```diff
@@ -656,7 +656,7 @@
   end
 
   create_table "request_logs", id: :uuid, default: -> { "uuid_generate_v4()" }, force: :cascade do |t|
-    t.uuid "account_id"
+    t.uuid "account_id", null: false
```

Comment:
This is out of sync. I didn't add this one to this PR. Likely from the previous PR. Will need to reset the schema.